src/ts/AppBar/index.tsx

```diff
@@ -50,14 +49,14 @@ export default class AppBarVG extends React.Component<any, any> {
     }
   }
 
-  public search(filter) { log('AppBar search change', filter) }
+  public search(filter) { log('AppBar search change', filter); }
 
   render() {
-    const line = this.state.visible && <div className="app-bar-line"></div>
-    const search = this.state.visible && <Search icon="search" filter={this.search} />
+    const line = this.state.visible && <div className="app-bar-line"></div>;
+    const search = this.state.visible && <Search icon="search" filter={this.search} />;
     const icon = this.state.visible
```

Comment:
if the only difference is in the className, perhaps we could just control the classname with this constant?

```
const icon = this.state.visible && ' border'
```
or even
```
<Link className={'title' + (this.state.visible && ' border')}" to="/">&#5809;</Link>
```

just make sure it does not render as the string literal `false`